.circleci/docker-compose.yml

```diff
@@ -0,0 +1,74 @@
+version: "3.5"
```

Comment:
Is it a copy from `scalar-samples`?
If so, I think it would be better to clone the repo and reuse it for maintainability.